src/MiniCover/Commands/Options/SolutionDirOption.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Microsoft.Extensions.CommandLineUtils;
+using System;
+using System.IO;
+
+namespace MiniCover.Commands.Options
+{
+    public class SolutionDirOption : IMiniCoverOption<DirectoryInfo>
```

Comment:
Can we keep using the workdir on this PR? I would like to avoid mixing breaking changes with the command and options refactor.

Reply:
Easier will be at this PR. Second has too many conflicts.